### PR TITLE
Update why-move-ecosystem.en-US.mdx

### PR DESCRIPTION
**Pull request by ren-ryder**
edits :)

### CODE REVIEW

File: pages/blog/why-move-ecosystem.en-US.mdx [hunk 1]
```diff
@@ -13,13 +13,13 @@ import PostHeader from "/components/blog/postHeader";
 
 As a proponent of the Move programming language, every time I promote Move to developers, I always encounter the same question: **What are the advantages of Move? Why Move?**
 
-It's like introducing your new lover to a friend - you always face similar questions. However, this question is not easy to answer. If we list the pros and cons one by one, there will always be people who question them. After all, the ecosystem of a new language is not yet mature, and the choice can only be based on its potential. Let me make a statement: **Move has the most potential programming language to build an ecosystem like Solidity, and even surpass Solidity**.
+Introducing your favorite topic to an outsider, you always face the same kind of questions. However, this question is not easy to answer. If we list the pros and cons one by one, there will always be people who question them. After all, the ecosystem of a new language is not yet mature, and the choice can only be based on its potential. Let me make a statement: **Move has the most potential of any programming language to build an ecosystem like Solidity, and even to surpass Solidity.**
```
Comment (geometryolife) [inline pages/blog/why-move-ecosystem.en-US.mdx:16]:
```suggestion
Introducing your favorite topic to an outsider, you always face the same kind of questions. However, this question is not easy to answer. If we list the pros and cons one by one, there will always be people who question them. After all, the ecosystem of a new language is not yet mature, and the choice can only be based on its potential. Let me make a statement: **Move has the most potential of any programming language to build an ecosystem like Solidity, and even to surpass Solidity**.
```

File: pages/blog/why-move-ecosystem.en-US.mdx [hunk 2]
```diff
@@ -293,9 +293,9 @@ This section illustrates a new way of combining things in Move based on linear t
 
 ## Expanding the Application Scenarios of Smart Contracts
 
-Move, originally designed as the smart contract programming language for the Libra.
+Move, originally designed as the smart contract programming language Libra.
```
Comment (geometryolife) [inline pages/blog/why-move-ecosystem.en-US.mdx:296]: Liibra is a blockchain.

**Review by geometryolife — COMMENTED** [review]
The review has been completed. @ren-ryder

File: pages/blog/why-move-ecosystem.en-US.mdx (hunk 2 above)
Comment (ren-ryder) [reply pages/blog/why-move-ecosystem.en-US.mdx:296]: Ah, good catch. Should be "for the Libra blockchain."

**Review by geometryolife — APPROVED** [review]

**Pull request merged**